indexing_utilities.cpp

```diff
@@ -293,7 +293,7 @@ namespace irods {
 
             static const auto double_quote = R"(")"s;
             const auto JOB_QUERY_STRING = fmt::format (R"(SELECT RULE_EXEC_ID WHERE RULE_EXEC_NAME like '%"{}"%')", unique_key);
-
+                                                              // ^^^^^^^^^^^^ - make this count(RULE_EXEC_ID) to save bandwidth ?
```

Comment:
if we only need the count, and not the values... then yes, would be faster

Reply:
done, count used now in query